### DISCUSSION TITLE
SCTPD service not getting started

### DISCUSSION THREAD

**Discussion by AaquilAli** [magma/magma#6019] [General]
Hi all
I am trying to test the magma core with openairinterface5g UE and ENB/GNB. But I am facing one isssue and the issue is that after SCTP INIT message send by the enb, magma core is sending ICMP error message with Destination unreachable (Protocol unreachable), verified this in the pcap.
After some debugging I have found that the sctpd process is not starting . When I hit the command "sudo service sctpd status" I get the below output.

[sctpd_status.zip](https://github.com/magma/magma/files/6276140/sctpd_status.zip)

Need help to resolve this issue.
Thanks in advance.

**Comment by ulaskozat** [comment]
could you paste the output for `dpkg -l | grep magma`?

**Reply by AaquilAli** [reply]
vagrant@magma-dev:~/magma/lte/gateway$ dpkg -l | grep magma
ii  magma-cpp-redis                           4.3.1.2-2                         amd64        Redis C++ client library
ii  magma-libfluid                            0.1.0.5-1                         amd64        Libfluid Openflow Controller

**Reply by ulaskozat** [reply]
I don't see magma or magma-sctpd here. You have not installed them it seems.

**Reply by AaquilAli** [reply]
I have used this link(https://docs.magmacore.org/docs/basics/quick_start_guide) to install the set up. After accessing the VBox using the command "vagrant ssh magma" , I hit the command "make run".  Looks like this command does not install the magma-sctpd. 
So how can i install magma or magma-sctpd . Please could you guide me on that.

**Reply by ulaskozat** [reply]
ah, sorry I forgot that this was vagrant vm.

**Reply by ulaskozat** [reply]
would you mind destroying your vm and redo the quick start guide (check if you receive any warnings/errors during the process)?

**Reply by AaquilAli** [reply]
I destroyed the VM and tried to redo everything . So while doing the "vagrant up magma ", the installation got failed with the following error . I am also attaching a log file.
[vagrant_up.log](https://github.com/magma/magma/files/6294145/vagrant_up.log)



magma_focal                : ok=124  changed=21   unreachable=0    failed=1    skipped=14   rescued=0    ignored=0

Monday 12 April 2021  05:06:37 +0000 (0:00:00.279)       0:10:14.509 **********
===============================================================================
magma : Extract Go tarball -------------------------------------------- 120.01s
dev_common : Setup FPM so we can build packages ------------------------ 66.82s
dev_common : Clone swagger-codegen repo -------------------------------- 63.97s
dev_common : Install Debian Magma gateway dev dependencies ------------- 36.14s
magma : Download golang tar -------------------------------------------- 21.57s
magma : Restart networking to bring up linux bridge -------------------- 17.44s
magma : Install OpenAirInterface (OAI) dependencies -------------------- 15.17s
pyvenv : Create virtualenv --------------------------------------------- 12.45s
magma : Install common Magma dependencies ------------------------------ 11.73s
magma : Copy magma service files --------------------------------------- 11.62s
pkgrepo : Add JFrog repo ----------------------------------------------- 11.23s
envoy : Add Getenvoy.io apt key ----------------------------------------- 9.05s
dev_common : Install ansible community collection ----------------------- 8.73s
magma : Install dnsmasq ------------------------------------------------- 7.90s
magma : Install debian Magma dependencies ------------------------------- 7.37s
magma : Install C code coverage analysis dependencies ------------------- 7.29s
dev_common : Setup build requirements for python sdk -------------------- 7.13s
magma : Specific Magma dependencies from backports ---------------------- 7.01s
envoy : Ensure required dependencies packages are installed. ------------ 6.95s
magma : Install gmock and gtest for C++ testing ------------------------- 6.79s
Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.

**Reply by AaquilAli** [reply]
@ulaskozat , I installed the package nlohmann, even after that  I am getting the same error. If this could be the reason for sctpd to fail?
